_quarto.yml

```
@@ -43,6 +43,9 @@ website:
         href: interpreters.qmd
       - text: "Help"
         href: troubleshooting.qmd
+    right:
+      - icon: github
+        href: https://github.com/posit-dev/positron
```

Since we're using an icon for the link (with no visible text label), let's include an aria-label:

```suggestion
    right:
      - icon: github
        aria-label: 'Link to Positron Github Repository'
        href: https://github.com/posit-dev/positron
```